src/__test__/matchMedia.mock.js

```diff
@@ -0,0 +1,16 @@
+// From https://jestjs.io/docs/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom
+// TO-DO: we mock 'matchMedia' in several files
```

Comment:
what is the "todo" action in here? To mock the rest too?

Reply:
We have lots of files with
```
  beforeEach(() => {
    Object.defineProperty(window, 'matchMedia', {
      writable: true,
      value: jest.fn().mockImplementation((query) => ({
        matches: false,
        media: query,
        onchange: null,
        addListener: jest.fn(), // deprecated
        removeListener: jest.fn(), // deprecated
        addEventListener: jest.fn(),
        removeEventListener: jest.fn(),
        dispatchEvent: jest.fn(),
      })),
    });
  });
```
This can be removed now

Reply:
that's great!